includes/admin/class-wp-job-manager-promoted-jobs-admin.php

```diff
@@ -209,7 +209,8 @@ public function handle_promote_job() {
 		$verify_endpoint_url = rest_url( '/wpjm-internal/v1/promoted-jobs/verify-token', 'https' );
 		$verify_endpoint_url = substr( $verify_endpoint_url, strlen( $site_url ) );
 
-		update_option( WP_Job_Manager_Promoted_Jobs_Status_Handler::USED_PROMOTED_JOBS_OPTION_KEY, true );
+		add_option( WP_Job_Manager_Promoted_Jobs_Status_Handler::CRON_INTERVAL_OPTION_KEY, 12 * HOUR_IN_SECONDS, '', false );
+		add_option( WP_Job_Manager_Promoted_Jobs_Status_Handler::WEBHOOK_INTERVAL_OPTION_KEY, 5 * MINUTE_IN_SECONDS, '', false );
```

Comment:
These values are literally the default that we had set on the feature branch. Except that now they're options and are controllable via HTTP Response Headers.